test/csharp/test_MshSnapinInfo.cs

```diff
@@ -5,20 +5,20 @@
 namespace PSTests
 {
     // Not static because a test requires non-const variables
-    [Collection("AssemblyLoadContext")]
     public class MshSnapinInfoTests
     {
         // Test that it does not throw an exception
-        [Fact]
+        [SkippableFact]
         public void TestReadRegistryInfo()
         {
+            Skip.IfNot(Platform.IsWindows);
             Version someVersion = null;
             string someString = null;
             PSSnapInReader.ReadRegistryInfo(out someVersion, out someString, out someString, out someString, out someString, out someVersion);
         }
 
         // PublicKeyToken is null on Linux
-        [Fact]
+        [SkippableFact]
```

Comment:
does this need a `Skip.IfNot(Platform.IsWindows)`?

Reply:
Added